packages/communication-react/review/communication-react.api.md

```diff
@@ -1120,7 +1122,11 @@ export const DEFAULT_COMPOSITE_ICONS: {
     OptionsCamera: JSX.Element;
     OptionsMic: JSX.Element;
     OptionsSpeaker: JSX.Element;
-    ParticipantItemMicOff: JSX.Element;
+    ParticipantItemMicOff: JSX.Element; /**
+    * Icons that can be overridden for {@link CallComposite}.
```

Comment:
@prprabhu-ms it's happening again!! 🤯

This comment is a) out of place and b) should never exist in this file.
This happened before in one of my local builds -- I manually removed it (it doesn't happen in the fresh CI build) - I couldn't get to the bottom of _why_ it happened

Reply:
Thanks for the call out. I manually removed it. Not sure why this is happening either.